.github/workflows/1_2_stable_extension_release.yml

```diff
@@ -1,14 +1,26 @@
-name: 1/2c. Bump versions for extension only (promotes patch branch to stable release)
+name: 1/2. Bump and release a stable extension version
```

Comment:
❓ So this workflow is intended to be only trigger manually if we want to do a "one off manual release" of the extension, right?

If yes: Can we make this clearer in the description/naming? 🤔